app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ApplicationRecord
+  has_many :event_atendees
```

Comment:
this association is not correct, but there are two possible ways to make it:
1- First solution without a need for a new model you can use
* [ ] User model `has_and_belongs_to_many :attended_events, class_name: "Event"`
* [ ] Event model `has_and_belongs_to_many :attendees, class_name: "User"`

2- and this one you can use: 
* [ ] join model is required, its name can be `EventsAttendation` or anything else
* [ ] EventsAttendation model `belongs_to :attendee, class_name: "User"`
* [ ] EventsAttendation model `belongs_to :attended_event, class_name: "Event"`
* [ ] User model `has_many :event_attendations`
* [ ] User model `has_many :attended_events, through: :event_attendations`
* [ ] Event model `has_many :event_attendations`
* [ ] Event model `has_many :attendees, through: :event_attendations`